pallets/configuration/src/lib.rs

```diff
@@ -71,6 +71,14 @@ pub struct HostConfiguration {
     pub collators_per_container: u32,
     // If this value is 0 means that there is no rotation
     pub full_rotation_period: u32,
+    // Like collators_per_container but for parathreads
+    pub collators_per_parathread: u32,
+    // TODO: use a less confusing name? this is for when we support assigning 1 collator to more than 1 parathread at
+    // the same time, how many parathreads.
+    pub parathreads_per_collator: u32,
+    // % of collators that we expect to be assigned to container chains. Affects fees.
+    // TODO: I guess container chains and parathreads will be counted together
```

Comment:
We will need to see this, perhaps the count will be parachain based and we assign a percentage to each parathread (like each parathread counts 0.25 of a parachain)